core/functions/fn_postChecks.sqf

```diff
@@ -1,15 +1,19 @@
 #include "..\script_macros.hpp"
 
-[{
-    !(GVAR(Modules) isEqualTo [])
-},{
+[{!(GVAR(Modules) isEqualTo [])},{
 	player createDiarySubject [QGVAR(Menu), "Olsen Framework"];
 
 	private _modules = "<font size='18'>Olsen Framework Modules</font><br/><br/>";
 
 	for "_i" from 0 to (count GVAR(Modules) - 1) step 1 do {
 		private _module = GVAR(Modules) select _i;
-		_modules = _modules + "<font size='16'>" + (_module select 0) + "</font><br/>Description: " + (_module select 1) + "<br/>by " + (_module select 2);
+        _module params ["_name", "_description", "_author", ["_version", 0, [0]]];
+        _version = if (_version isEqualTo 0) then {
```

Comment:
A good way to deal with versions of modules that haven't been upgraded yet I guess, nice and clear to the user.